test/glue/pulp/repository_test.rb

```diff
@@ -284,6 +287,32 @@ def test_ostree_pulp_update_needed?
       ostree_repo.save!
       assert ostree_repo.pulp_update_needed?
     end
+
+    def test_pulp_checksum_type
+      repo = katello_repositories(:fedora_17_x86_64)
+
+      repo.stubs(:pulp_repo_facts)
+      repo.stubs(:distributors).returns([{
+        "distributor_type_id" => Runcible::Models::YumDistributor.type_id
+      }])
+
+      assert_nil repo.pulp_checksum_type
+
+      repo.stubs(:pulp_repo_facts).returns({"scratchpad" => {"checksum_type" => SHA1}})
+      repo.stubs(:distributors).returns([{
+        "config" => {},
+        "distributor_type_id" => Runcible::Models::YumDistributor.type_id
+      }])
+
+      assert_equal repo.pulp_checksum_type, SHA1
+
+      repo.stubs(:distributors).returns([{
+        "config" => {"checksum_type" => SHA256},
+        "distributor_type_id" => Runcible::Models::YumDistributor.type_id
+      }])
+
+      assert_equal repo.pulp_checksum_type, SHA256
+    end
```

Comment:
You stated as a checkbox: "create a test for pulp_checksum_type so if the API changes in the future, we can easily detect it"  But these tests don't actually do that.  I'm okay with the tests as is, just want to make sure its clear these don't do that.

Reply:
You are correct! :) I think this test is still useful, so maybe I'll add an additional VCR test.